src/View/Cell.php

```diff
@@ -176,6 +176,17 @@ public function render($template = null)
         }
 
         $render = function () use ($template) {
+            try {
+                $reflect = new ReflectionMethod($this, $this->action);
+                $reflect->invokeArgs($this, $this->args);
```

Comment:
IMHO avoiding invoking the action is more desirable than having the ability to change template from the cell action. You can always create multiple action and put the common stuff in a protected method.